src/components/Home/HardwareCard/HardwareCard.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC } from "react";
+import { useTranslation } from "react-i18next";
+import { HardwareInfo } from "../../../models/hardware-info";
+import LoadingBox from "../../Shared/LoadingBox/LoadingBox";
+
+const PI_NUM_CORES = 4;
+
+export type Props = {
+  hardwareInfo: HardwareInfo | null;
+};
+
+export const HardwareCard: FC<Props> = ({ hardwareInfo }) => {
+  const { t } = useTranslation();
+
+  if (!hardwareInfo) {
+    return (
+      <div className="h-full w-1/2">
+        <LoadingBox />
+      </div>
+    );
+  }
+
+  const {
+    cpu_overall_percent,
+    temperatures_celsius,
+    disks,
+    vram_usage_percent,
+  } = hardwareInfo;
+
+  const system_temp = temperatures_celsius.system_temp;
+
+  const cpuPercent = (cpu_overall_percent / PI_NUM_CORES) * 100;
+
+  const mainHDD = disks.find((disk) => disk.device === "/");
+
+  // byte => gigabyte converstion
+  const hddUsedGB = mainHDD?.partition_used_bytes
+    ? (mainHDD.partition_used_bytes / 1024 / 1024 / 1024).toFixed(2)
+    : "-";
+
+  const hddFreeGB = mainHDD?.partition_total_bytes
+    ? (mainHDD.partition_total_bytes / 1024 / 1024 / 1024).toFixed(2)
+    : "-";
+
+  let hddUsedPercent: string = "-";
+
+  if (!isNaN(+hddUsedGB) && !isNaN(+hddFreeGB)) {
+    hddUsedPercent = ((+hddUsedGB / +hddFreeGB) * 100).toFixed(2);
+  }
+
+  return (
+    <div className="bd-card mt-10 w-full transition-colors lg:mt-0 lg:ml-2 lg:w-1/2">
+      <div className="flex items-center text-lg font-bold">
+        {t("hardware.header")}
+      </div>
+      <article className="flex flex-row overflow-hidden py-4">
```

Comment:
nit: Don't know how this screen looks like but acrticle/h6/div might be a bit too much. Mayeb it's a `ul` or a `dl`? But non-blocking.

Reply:
Will look into it